cri/firecracker/service.go

```diff
@@ -216,5 +218,17 @@ func getEnvVal(key string, config *criapi.ContainerConfig) (string, error) {
 	}
 
 	return "", errors.New("failed to provide non empty guest image in user container config")
+}
+
+// (Key, Value) pair is mapped to a 'Key=Value' entry.
+func mapKeyValueToStringArray(envVariables []*criapi.KeyValue) []string {
+	result := make([]string, len(envVariables))
+
+	for _, kv := range envVariables {
+		env := fmt.Sprintf("%s=%s", kv.GetKey(), kv.GetValue())
+		result = append(result, env)
+	}
 
+	log.Infof("Converted '%v' to '%v'\n", envVariables, result)
```

Comment:
Debugf.

Reply:
Fixed

Reply:
Also added the changes to `vhive/cri/gvisor/coordinator.go:98`.